jsonchecker/__init__.py

```diff
@@ -42,7 +42,8 @@ def check_directory(self, directory):
         else:
             files = [directory]
         for fname in files:
-            if fname.startswith('.'):
+            first_directory = fname.split(os.path.sep)[0]
```

Comment:
probably better to use `os.path.abspath` on all of the names given in the command line options.